{"status": "broken", "name": "Ignore space before colon", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "LabelsAcceptance"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486031314, "historyId": "bdfe5811d0f40c3bc206af04b3c1ab03", "start": 1578486031089, "steps": [{"status": "passed", "start": 1578486031091, "stop": 1578486031304, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486031305, "stop": 1578486031308, "name": "And having executed", "attachments": [{"source": "70a123cf-537d-49bb-a4ba-7a55a7fbec17-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486031308, "stop": 1578486031310, "name": "When executing query", "attachments": [{"source": "a5310581-d19e-48d8-b5e5-e48fe54db304-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "e582ee0d-feae-4999-bf19-6e4b0a8a95cd-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486031312, "start": 1578486031311, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support set labels\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486031313, "stop": 1578486031313, "name": "And the side effects should be", "attachments": [{"source": "c4b25ed8-369e-4ed5-a29b-2543a8623dfb-attachment.csv", "type": "text/csv", "name": ".table"}]}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support set labels\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "b4f7ed3a-33d7-4f6d-a6a2-526724ba17f3"}